src/main/java/edu/harvard/iq/dataverse/api/BuiltinUsers.java

```diff
@@ -105,7 +108,29 @@ public Response create(BuiltinUser user, @PathParam("password") String password,
         return internalSave(user, password, key);
     }
     
+    /**
+     * Created this new endpoint to resolve issue #6915, optionally preventing 
+     * the email notification to the new user on account creation by adding 
+     * "false" as the third path parameter.
+     *
+     * @param user
+     * @param password
+     * @param key
+     * @param sendEmailNotification
+     * @return
+     */
+    @POST
+    @Path("{password}/{key}/{sendEmailNotification}")
+    public Response create(BuiltinUser user, @PathParam("password") String password, @PathParam("key") String key, @PathParam("sendEmailNotification") Boolean sendEmailNotification) {
```

Comment:
sendEmailNotification appears to be a path parameter rather than a query parameter, as documented above. @RightInTwo can you provide a full curl example?

Reply:
Hi @pdurbin, we added both ways (query and path), just as the create() function is implemented for both currently. The docs at http://guides.dataverse.org/en/latest/api/native-api.html#assign-default-role-to-user-creating-a-dataset-in-a-dataverse seem to omit the path variant completely, which is why we left it out as well.

Reply:
Oh, I get it. The query parameter version is called "save". Thanks.